Bottom: 06ef4166b818943012b3d41005acf03d4d9ab6a3
Top:    b634152e593147dcc9ad62930f64ffc29ed3b560
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-07-04 00:05:41 +0100

Refresh of constify-name-argument-to

---

diff --git a/src/hid/gtk/gui-utils.c b/src/hid/gtk/gui-utils.c
index 4e1bf60..887128f 100644
--- a/src/hid/gtk/gui-utils.c
+++ b/src/hid/gtk/gui-utils.c
@@ -617,7 +617,7 @@ ghid_scrolled_selection (GtkTreeView * treeview, GtkWidget * box,
 }
 
 GtkWidget *
-ghid_notebook_page (GtkWidget * tabs, char *name, gint pad, gint border)
+ghid_notebook_page (GtkWidget * tabs, const char *name, gint pad, gint border)
 {
   GtkWidget *label;
   GtkWidget *vbox;
diff --git a/src/hid/gtk/gui.h b/src/hid/gtk/gui.h
index afe3501..175f145 100644
--- a/src/hid/gtk/gui.h
+++ b/src/hid/gtk/gui.h
@@ -412,7 +412,7 @@ GtkWidget *ghid_framed_vbox_end (GtkWidget * box, gchar * label,
 GtkWidget *ghid_category_vbox (GtkWidget * box, const gchar * category_header,
 			       gint header_pad, gint box_pad,
 			       gboolean pack_start, gboolean bottom_pad);
-GtkWidget *ghid_notebook_page (GtkWidget * tabs, char *name, gint pad,
+GtkWidget *ghid_notebook_page (GtkWidget * tabs, const char *name, gint pad,
 			       gint border);
 GtkWidget *ghid_framed_notebook_page (GtkWidget * tabs, char *name,
 				      gint border, gint frame_border,
